src/integrationtests/java/com/aws/greengrass/integrationtests/SyncTest.java

```diff
@@ -432,6 +436,71 @@ void GIVEN_synced_shadow_WHEN_local_update_THEN_cloud_updates(Class<?extends Bas
                 any(software.amazon.awssdk.services.iotdataplane.model.UpdateThingShadowRequest.class));
     }
 
+    @ParameterizedTest
+    @ValueSource(classes = {RealTimeSyncStrategy.class, PeriodicSyncStrategy.class})
+    void GIVEN_synced_shadow_WHEN_multiple_local_updates_THEN_cloud_updates(Class<?extends BaseSyncStrategy> clazz, ExtensionContext context) throws IOException,
```

Comment:
Can we add a test for LocalShadowUpdate as well?

Also, can you check if you can intertwine a Cloud update and a local update in a similar manner in another test?

Reply:
added